.pre-commit-config.yaml

```diff
@@ -23,3 +23,8 @@ repos:
     - id: rst-backticks
     - id: rst-directive-colons
     - id: rst-inline-touching-normal
+
+ - repo: https://github.com/eos/pre-commit-hooks
+   rev: v1.0.0
+   hooks:
+    - id: eos-test-check-argument-order
```

Comment:
Can you extend the scope of your hook to ``TEST_CHECK_RELATIVE_ERROR`` and ``TEST_CHECK_RELATIVE_ERROR_C``?

Reply:
Yes, let's move that over to ``eos/pre-commit-hooks``.